scripts/kube-init.sh

```diff
@@ -30,6 +30,11 @@ trap "clean_exit" EXIT
 # Switch off SE-Linux
 setenforce 0
 
+# Mount root to fix dns issues
+# Define $HOME since somehow this is not defined
+export HOME=/home/travis
+sudo mount --make-rshared /
```

Comment:
Which dns issue does this solve?

Reply:
Frankly I am not sure. I’ve seen others using it and we had disable dns in our previous script. I think ```kubedns``` sometimes does not like the CI environments.

Reply:
Read https://github.com/LiliC/travis-minikube/blob/minikube-30-kube-1.12/.travis.yml. SGTM